### PR TITLE
changes for getting standard local currency info from IOS billing plugin

### PR DESCRIPTION
**Pull request by vivekb91**

### CODE REVIEW

**Review by rampr — APPROVED** [review]

File: ios/BillingPlugin.mm [hunk 1]
```diff
@@ -88,8 +89,7 @@ - (void) failedTransaction: (SKPaymentTransaction *)transaction {
 										  [NSNull null],@"token",
 										  errorCode,@"failure",
 										  nil]];
-
-    [[SKPaymentQueue defaultQueue] finishTransaction: transaction];
+	[[SKPaymentQueue defaultQueue] finishTransaction: transaction];
```
Comment (bijoshtj) [inline ios/BillingPlugin.mm]: indentation

File: ios/BillingPlugin.mm [hunk 2]
```diff
@@ -116,6 +116,11 @@ - (void)productsRequest:(SKProductsRequest *)request didReceiveResponse:(SKProdu
 			if (sku != nil && [sku hasPrefix:self.bundleID]) {
 				sku = [sku substringFromIndex:len];
 				[prices setObject:formattedPrice forKey:sku];
+
+        NSDictionary *priceDetails = [NSDictionary dictionaryWithObjectsAndKeys:
+                                              product.price,@"localPrice",
+                                              product.priceLocale.currencyCode,@"currencyCode", nil];
+        [self.productDetails setObject:priceDetails forKey:sku];
```
Comment (bijoshtj) [inline ios/BillingPlugin.mm]: indentations..

File: ios/BillingPlugin.mm (hunk 1 above)
Comment (vivekb91) [reply ios/BillingPlugin.mm]: This indentation is correct

**Review by bijoshtj — APPROVED** [review]

**Comment by rampr** [comment]
@vivekb91  can we close this if this isn't needed ?

**Pull request closed without merging**